script/Ciel/Day 3/03_01_CIEL03_2_3.txt

```diff
@@ -352,7 +353,7 @@ There have been eight victims already, so the police will probably catch the cul
 -- Page 85, Offset 5051. Mods: @n.
 -- 「もうっ。そんなコトを言っていると寝首をかかれますよ。
 // Lit. expression 'to cut the head off a sleeping person'
-"Oh, no. If you keep talking like that, you'll get attacked while your guard is down.
+"Come on. If you keep talking like that, you'll get attacked while your guard is down.
```

Comment:
be attacked

Reply:
can't say I agree. "get attacked" sounds more natural to me